app/services/feature_flag.rb

```diff
@@ -26,22 +26,32 @@ def feature
     FeatureFlag.new(name: name)
   }.with_indifferent_access.freeze
 
-  def self.activate(feature_name)
+  def self.activate(feature_name, **opts)
     raise unless feature_name.in?(FEATURES)
 
-    sync_with_database(feature_name, true)
+    if opts.key?(:for).present?
+      sync_with_database_with_object(feature_name, opts[:for], true)
+    else
+      sync_with_database(feature_name, true)
+    end
   end
 
-  def self.deactivate(feature_name)
+  def self.deactivate(feature_name, **opts)
```

Comment:
Do we need a way to easily deactivate for all? E.g. if we start rollout of a feature that is so broken we want to turn it off

Reply:
Good question. Another one I had in mind is if we would like to globally activate the feature with just a few exceptions? But that is much more edgy.

For now I'll add `for: :all` option that destroys all the selected objects as well.